### ISSUE TITLE
Maximize/unmaximize windows animation breaks in Epiphany when a website is loaded 

### ISSUE THREAD

**Issue by danilomataveli** [elementary/browser#38]
first, congratulations to everyone for the release of elementary OS 6, a great advance and an amazing job.

i am opening this issue because i didn't find it among the ones already open, if i am wrong, please notify me.

what happened?

when maximizing or unmaximize the Epiphany window with a loaded site, it crashes, displaying a non-standard behavior. The window splits in two pieces.

this does not happen when a site is not loaded, and as far as I was able to test, it doesn't happen with the other standard system apps.

https://user-images.githubusercontent.com/69722851/128935694-2671d608-a327-4b8d-8f47-9d4728b523a6.mp4

![Captura de tela de 2021-08-10 16-59-07](https://user-images.githubusercontent.com/69722851/128935766-ed0f554b-ae45-452e-b29c-ea14d1a52400.png)

how to reproduce?

1. open Epiphany
2. load a site
3. maximize/unmaximize the Epiphany window

System and Hardware info

![Captura de tela de 2021-08-10 18-08-01](https://user-images.githubusercontent.com/69722851/128935823-3f155a2a-26c8-4e57-9447-d6c6cfe3c516.png)
![Captura de tela de 2021-08-10 18-07-57](https://user-images.githubusercontent.com/69722851/128935830-0e2a575e-7e9d-406c-a4c8-8e8e39b49b16.png)

**Comment by janxkoci** [comment]
But does it stay like that, in two pieces? I can see it happening during the transition (for years actually), but it's always a matter of a second or two, before the window snaps into the right position. It never stayes like that for me.

**Comment by danilomataveli** [comment]
Yeah, it's only during the transitions. Thanks for the observation.

**Comment by janxkoci** [comment]
I suspect this will be more related to the web content rendering, meaning the WebkitGTK engine, not Epiphany itself. I would be surprised if it wasn't reported on Webkit bugzilla already, but I will check later, now I'm a bit busy.

**Comment by danilomataveli** [comment]
It's possible. I'm new whit those technical terms and details. Wish I could help in other directions. Tell me if you find anything.
